integration/cli-tests/src/test/java/org/wso2/carbon/esb/cli/CliSequenceTestCase.java

```
@@ -0,0 +1,153 @@
+            log.info(cliSampleSeq + " Sequence Found");
+
+        } catch (IOException e) {
+            log.info("Exception = " + e.getMessage());
+        } finally {
+            reader.close();
+        }
+    }
+
+    /**
+     * Get information about single Sequence
+     */
+
+    @Test
+    public void miShowEndpointTest() throws IOException {
+
+        try{
+            ProcessBuilder builder = new ProcessBuilder(miBuildFilePath.getCanonicalPath(), "sequence" , "show", cliTestSeq);
+            Process process = builder.start();
+            reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
+
+            List<String> lines = new ArrayList();
+            while ((line = reader.readLine()) != null) {
+                lines.add(line);
+            }
+            Assert.assertTrue(lines.stream().anyMatch(str -> str.trim().contains(cliTestSeq)),cliTestSeq +" Sequence not found");
+            log.info(cliTestSeq + " Sequence Found");
+
+        } catch (IOException e) {
+            log.info("Exception = " + e.getMessage());
+        } finally {
+            reader.close();
+        }
+    }
+
+    /**
+     * Test un-deployed Sequence
+     */
+
+    @Test
+    public void miShowEndpointNotFoundTest() throws IOException {
+
+        try{
+            ProcessBuilder builder = new ProcessBuilder(miBuildFilePath.getCanonicalPath(), "sequence" , "show", "CLITestSequence");
+            Process process = builder.start();
+            reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
+
+            List<String> lines = new ArrayList();
+            while ((line = reader.readLine()) != null) {
+                lines.add(line);
+            }
+            Assert.assertTrue(lines.stream().anyMatch(str -> str.trim().contains("Sequence 404 Not Found")),"Sequence 404 Not Found");
+            log.info("Sequence 404 Not Found");
+
+        } catch (IOException e) {
```

We need to throw the raised exceptions.